style.css

```diff
@@ -157,18 +158,420 @@ button {
   padding-left: 0;
 }
 
-.icon:hover,
-a:hover,
+.project-grid {
+  display: grid;
+  box-sizing: border-box;
+  border-radius: 5%;
+  margin: auto;
+  padding-top: 6rem;
+  background: #fff;
+  margin-left: 1.5rem;
+  margin-right: 1.5rem;
+}
+
+.project-card {
+  display: flex;
+  flex-direction: column;
+  border-radius: 2rem;
+  box-sizing: border-box;
+  margin-top: 5rem;
+  border: 1em;
+  background-color: #fff;
+}
+
+.project-img > img {
+  max-width: 95%;
+  max-height: 100%;
+  display: block;
+  margin-left: 0.7rem;
+  margin-top: 1rem;
+}
+
+.project-card > h2 {
+  padding-left: 0.5em;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  color: #172b4d;
+  justify-items: baseline;
+}
+
+.overhead {
+  max-width: 100%;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  position: relative;
+  top: -1em;
+  left: -1.5em;
+  padding-right: 0.5em;
+}
+
+.overhead-1 {
+  color: #344563;
+  list-style-type: none;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+}
+
+.overhead :not(.overhead-1) {
+  color: #7a869a;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+}
+
+.project-detail {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1rem;
+  line-height: 1.7rem;
+  color: #344563;
+  padding-left: 1em;
+  position: relative;
+  top: -1em;
+}
+
+.built-with {
+  display: flex;
+  justify-content: space-between;
+  max-width: 80%;
+  list-style-type: none;
+  position: relative;
+  left: -1.5em;
+}
+
+.build {
+  background-color: #ebebff;
+  width: 100%;
+  text-align: center;
+  height: 2.5rem;
+  border-radius: 1.1em;
+  padding: 0.8em;
```

Comment:
- @AliRezaBmeDu I noticed on the tags for the projects, the rounded borders are looking different from what are on the Figma design, and its because you made use of `1.1em`,  you should try following the Figma guidelines, on the figma it is `border-radius: 8px;`. Please adjust this.

##### Figam Design 
![image](https://github.com/AliRezaBmeDu/Portfolio-Website/assets/58771507/83e54f09-982e-491c-8e00-ff04bdffde2e)

##### Your design 
![image](https://github.com/AliRezaBmeDu/Portfolio-Website/assets/58771507/c4ab109d-23b5-4f29-85aa-9ab58359d7ce)